tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+from raiden_libs.test.fixtures.address import *  # noqa
+from raiden_libs.test.fixtures.contracts import *  # noqa
+from raiden_libs.test.fixtures.client import *  # noqa
+from raiden_libs.test.fixtures.web3 import *  # noqa
+
+from .fixture_overwrites import *  # noqa
+
+
+def pytest_addoption(parser):
```

Comment:
let's assume tests work with `eth-tester` only, until proved otherwise

Reply:
Ok, I'll remove this